Can't install on MacOS

## 🐛 Bug

I followed all of the install instructions and am getting the following error when I run `pip install nle`

```
Building wheel for nle (PEP 517) ... error
  ERROR: Command errored out with exit status 255:
   command: /Users/trentonbricken/anaconda3/envs/nle/bin/python /Users/trentonbricken/anaconda3/envs/nle/lib/python3.8/site-packages/pip/_vendor/pep517/_in_process.py build_wheel /var/folders/1q/6pbw30m128z5zbp5kfft7rn40000gn/T/tmps216ak48
       cwd: /private/var/folders/1q/6pbw30m128z5zbp5kfft7rn40000gn/T/pip-install-6l1fgazr/nle
  Complete output (46 lines):
  fatal: not a git repository (or any of the parent directories): .git
  Building wheel nle-0.1.1
  Successfully built fbs files from win/rl
  Found existing nle/fbs -- Removing old package
  Copying win/rl/nle/fbs to nle/fbs
  running bdist_wheel
  running build
  Building and installing NetHack...
  ./mkmkfile.sh: line 14: ../../doc/Makefile: No such file or directory
  ./mkmkfile.sh: line 15: ../../doc/Makefile: No such file or directory
  ./mkmkfile.sh: line 16: ../../doc/Makefile: No such file or directory
  ./mkmkfile.sh: line 17: ../../doc/Makefile: No such file or directory
  ./mkmkfile.sh: line 18: ../../doc/Makefile: No such file or directory
  ./mkmkfile.sh: line 19: ../../doc/Makefile: No such file or directory
  ./mkmkfile.sh: line 21: ../../doc/Makefile: No such file or directory
  ./mkmkfile.sh: line 22: ../../doc/Makefile: No such file or directory
  ./mkmkfile.sh: line 23: ../../doc/Makefile: No such file or directory
  ./mkmkfile.sh: line 26: ../../doc/Makefile: No such file or directory
  ./mkmkfile.sh: line 27: ../../doc/Makefile: No such file or directory
  ./mkmkfile.sh: line 28: ../../doc/Makefile: No such file or directory
  ./mkmkfile.sh: line 30: ../../doc/Makefile: No such file or directory
  ./mkmkfile.sh: line 31: ../../doc/Makefile: No such file or directory
  ./mkmkfile.sh: line 32: ../../doc/Makefile: No such file or directory
  ./mkmkfile.sh: line 33: ../../doc/Makefile: No such file or directory
  ./mkmkfile.sh: line 34: ../../doc/Makefile: No such file or directory
  ./mkmkfile.sh: line 35: ../../doc/Makefile: No such file or directory
  ./mkmkfile.sh: line 37: ../../doc/Makefile: No such file or directory
  ./mkmkfile.sh: line 38: ../../doc/Makefile: No such file or directory
  ./mkmkfile.sh: line 39: ../../doc/Makefile: No such file or directory
  ./mkmkfile.sh: line 42: ../../doc/Makefile: No such file or directory
  ./mkmkfile.sh: line 43: ../../doc/Makefile: No such file or directory
  ( cd src ; /Applications/Xcode.app/Contents/Developer/usr/bin/make )
  touch ../src/config.h-t
  gcc -Wall -Wextra -Wno-missing-field-initializers -Wimplicit -Wreturn-type -Wunused -Wformat -Wswitch -Wshadow -Wwrite-strings -DGCC_WARN -g -I../include -DNOCLIPPING -DNOMAIL -DNOTPARMDECL -DHACKDIR=\"/Users/trentonbricken/anaconda3/envs/nle/lib/games/nethack\" -DDEFAULT_WINDOW_SYS=\"tty\" -DDLB -c monst.c
  In file included from /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk/usr/include/_stdio.h:68,
                   from /usr/local/lib/gcc/x86_64-apple-darwin19/9.2.0/include-fixed/stdio.h:78,
                   from ../include/global.h:9,
                   from ../include/config.h:583,
                   from monst.c:6:
  /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk/usr/include/sys/cdefs.h:264:39: error: macro "__has_attribute" requires an identifier
    264 | #if __has_attribute(warn_unused_result)
        |                                       ^
  make[1]: *** [monst.o] Error 1
  make: *** [nethack] Error 2
  Generated NetHack makefiles with hints/macosx.
  Failed to build NetHack
  ----------------------------------------
  ERROR: Failed building wheel for nle
Failed to build nle
ERROR: Could not build wheels for nle which use PEP 517 and cannot be installed directly`
```
## Environment

MacOS Catalina, 10.15.4

Edit (@edran): Fixed trace formatting.

--- a/sys/unix/hints/macosx
+++ b/sys/unix/hints/macosx
@@ -47,7 +47,7 @@ GAMEGRP  = $(GAMEUID)
 
 
 #CC=gcc -W -Wimplicit -Wreturn-type -Wunused -Wformat -Wswitch -Wshadow -Wcast-qual -Wwrite-strings -DGCC_WARN
-CC=gcc -Wall -Wextra -Wno-missing-field-initializers -Wimplicit -Wreturn-type -Wunused -Wformat -Wswitch -Wshadow -Wwrite-strings -DGCC_WARN
+CC=clang -Wall -Wextra -Wno-missing-field-initializers -Wimplicit -Wreturn-type -Wunused -Wformat -Wswitch -Wshadow -Wwrite-strings -DGCC_WARN
 
 #
 # You shouldn't need to change anything below here.

